content/en/continuous_integration/faq/do-we-charge-for-bot-no-reply-committers.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Does Datadog charge for bot or no-reply committers with GitHub.com email addresses? 
+further_reading:
+- link: "/continuous_integration/"
+  tag: "Documentation"
+  text: "Learn about CI Visibility"
+---
+
+[Fill in the description.]
```

Comment:
```suggestion
Datadog does not charge for bot or actions made in the GitHub UI. These types of committers are excluded from billing calculations.
```

We should be more precise IMO, because people would expect to never have no-reply committers, whereas this is not the case.